libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt

```diff
@@ -107,6 +126,7 @@ internal object MapboxNavigationTelemetry :
             }
         }
     private val dynamicValues = DynamicSessionValues()
+    private val dynamicFreeDriveValues = DynamicFreeDriveValues()
     private var locationEngineNameExternal: String = LocationEngine::javaClass.name
```

Comment:
If Proguard is enabled (`minifyEnabled true`) this is a problem 👀 

```
{
  applicationState: 'Foreground',
  audioType: 'headphones',
  batteryLevel: 100,
  batteryPluggedIn: true,
  connectivity: 'Unknown',
  created: '2020-11-11T15:16:33.121Z',
  event: 'navigation.freeDrive',
  eventType: 'start',
  eventVersion: 7,
  location: {
    altitude: 121.5999984741211,
    course: 278.6067,
    horizontalAccuracy: 3.98,
    lat: 39.1161984,
    lng: -77.1692502,
    speed: 0.32502484,
    timestamp: '1605107791000',
    verticalAccuracy: 2
  },
  locationEngine: 'd.b.a.a.d.g',
  percentTimeInForeground: 99,
  percentTimeInPortrait: 100,
  screenBrightness: 11,
  sessionIdentifier: '3a35c325-c168-4515-91c2-bcfc5ae1eabf',
  simulation: false,
  startTimestamp: '2020-11-11T10:16:33.113-0500',
  volumeLevel: 0,
  received: '2020-11-11T15:16:47.452Z',
```

`locationEngine: 'd.b.a.a.d.g',` this is a problem for all of the events, when minification is on.

Reply:
let's try to fix it with
`@Keep val locationEngine: LocationEngine,` in `NavigationOptions`
if it doesn't help, let's cut another issue.

Reply:
created a separate issue